beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -1875,14 +1875,14 @@ pub static DATA_AVAILABILITY_OVERFLOW_STORE_CACHE_SIZE: LazyLock<Result<IntGauge
 pub static DATA_AVAILABILITY_RECONSTRUCTION_TIME: LazyLock<Result<Histogram>> =
     LazyLock::new(|| {
         try_create_histogram(
-            "data_availability_reconstruction_time_seconds",
+            "beacon_data_availability_reconstruction_time_seconds",
             "Time taken to reconstruct columns",
         )
     });
 pub static DATA_AVAILABILITY_RECONSTRUCTED_COLUMNS: LazyLock<Result<IntCounter>> =
     LazyLock::new(|| {
         try_create_int_counter(
-            "data_availability_reconstructed_columns_total",
+            "beacon_data_availability_reconstructed_columns_total",
             "Total count of reconstructed columns",
         )
     });
```

Comment:
Could you also update the 4 data column metrics below to include the prefix too?

Reply:
Found 3 `kzg_data_column` metrics below. What about `data_availability_cache_size` metrics above?